### PR TITLE
Use refactored networker

### PR DESCRIPTION
**Pull request by majst01**
This PR is required to make use of the refactored metal-networker, and pass ci again.

@GrigoriyMikhalkin we should take this first and then the DNS stuff. WDYT ?

### CODE REVIEW

File: pkg/network/network.go [hunk 1]
```diff
@@ -25,7 +26,13 @@ var templates embed.FS
 // ReconcileNetwork reconciles the network settings for a firewall
 // in the current stage it only changes the FRR-Configuration when network prefixes or FRR template changes
 func ReconcileNetwork(f firewallv1.Firewall, log logr.Logger) (bool, error) {
-	kb := netconf.NewKnowledgeBase(MetalKnowledgeBase)
+	// FIXME use zapr ?
```
Comment (GrigoriyMikhalkin) [inline pkg/network/network.go:29]: I'm totally agree with this. But `zapr` just wraps `zap` logger into `logr.Logger` interface. So we would have to change the `config` struct on the `metal-networker` side.

File: pkg/network/network.go [hunk 2]
```diff
@@ -25,7 +26,13 @@ var templates embed.FS
...
+	zlog, _ := zap.NewProduction()
+
+	kb, err := netconf.New(zlog.Sugar(), MetalKnowledgeBase)
```
Comment (GrigoriyMikhalkin) [inline pkg/network/network.go:32]: `MetalKnowledgeBase` -- should we rename it to `MetalNetworkerConfig`?

Comment (majst01) [reply pkg/network/network.go:32]: Yes

Comment (majst01) [reply pkg/network/network.go]: done

**Review by GrigoriyMikhalkin — APPROVED** [review]
LGTM

**Pull request merged**